server/tests/auth.test.js

```diff
@@ -0,0 +1,62 @@
+import request from 'supertest-as-promised';
+import httpStatus from 'http-status';
+import jwt from 'jsonwebtoken';
+import chai, { expect } from 'chai';
+import app from '../../index';
+
+const config = require('../../config/env');
+const should = require('chai').should();
+
+chai.config.includeStack = true;
+
+describe('## AUTH APIs', () => {
+  const user = {
+    username: 'react',
+    password: 'express'
+  };
+  let jwtToken;
+
+  describe('# POST /api/auth/login', () => {
+    it('should get (valid) JWT token', (done) => {
+      request(app)
+        .post('/api/auth/login')
+        .send(user)
+        .expect(httpStatus.OK)
+        .then((res) => {
+          should.exist(res.body.token);
+          jwt.verify(res.body.token, config.jwtSecret, (err, decoded) => {
+            should.not.exist(err);
```

Comment:
As in line with the above comment, can you replace this with `expect` assertion ?